gemstash.gemspec

```diff
@@ -28,7 +28,12 @@ you push your own private gems as well."
   spec.executables   = spec.files.grep(%r{^exe/}) {|f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activesupport", "~> 4.2"
+  if RUBY_VERSION > "2.3"
```

Comment:
Yes, sadly.

Perhaps we could make the check match 2.4-only, since that's the one which fails hard on older activesupport.

Like so:

```ruby
if RUBY_VERSION > "2.4"
```